app/controllers/device_stories_controller.rb

```diff
@@ -4,13 +4,26 @@ class DeviceStoriesController < ApplicationController
   has_scope :order
 
   def index
-    @device_stories = apply_scopes(DeviceStory)
-      .page(params[:page])
-      .per(params[:per_page])
+    @device_stories = if params[:search].blank?
+                        apply_scopes(DeviceStory).page(params[:page]).per(params[:per_page])
+                      else
+                        @search_term = params[:search].downcase
+                        get_elastic_searched_table(@search_term)
+                      end
   end
 
   def show
     @device_story = DeviceStory.find(params[:id])
     respond_with @device_story
   end
+
+  def get_like_searched_table(search_term)
+    apply_scopes(DeviceStory).where('lower(device_urn) LIKE :search OR lower(custodian_name) LIKE :search', search: "%#{search_term}%")
```

Comment:
I wouldn't expect you'd need both `LIKE` and elasticsearch here.

Once the data is in elasticsearch you should be able to search there. Plus text fields get analyized automatically so no need for cast/lower/etc

Reply:
<img width="1904" alt="Screen Shot 2020-11-13 at 6 58 36" src="https://user-images.githubusercontent.com/690/99038606-1cfe1580-25c9-11eb-9e13-0d570acb4526.png">

Seems to be working fine locally. So it should show up on prod once deployed as well.

We _might_ need to update the permissions to post, but that shouldn't be a problem.